examples/n2hp_cube_example.py

```diff
@@ -23,7 +23,7 @@
 spc.Registry.add_fitter('n2hp_vtau',pyspeckit.models.n2hp.n2hp_vtau_fitter,4)
 
 # Get a measurement of the error per pixel
-errmap = spc.slice(20, 28, unit='km/s').cube.std(axis=0)
+errmap = spc.slice(20, 28, units='km/s').cube.std(axis=0)
```

Comment:
We can allow `unit` or `units`, I think, to avoid having to make this sort of change.  But this is an aside, low-priority...